chore: Update dependencies

naterminch opened the pull request:
update 
"kysely": "^0.27.2", "@planetscale/database": "^1.14.0"

to fix type issue of mysql adapter.

jacobwgillespie commented:
@naterminch hey, thanks for this - what type issue are you seeing? The dependencies you've updated here are `devDependencies` and won't exist in the published npm package, so I'm not sure that this PR will actually fix the error.

naterminch commented:
Its a minor issue. After i updated the dependencies, i got a type error here.
```
"@planetscale/database": "^1.15.0",
    "kysely": "^0.27.2",
    "kysely-planetscale": "^1.4.0",
```
 
![Screenshot 2024-02-05 at 7 57 12 PM](https://github.com/depot/kysely-planetscale/assets/32100399/4a4966d0-0de8-4a4a-9423-2feef13fede3)

I updated the dependency, built it locally and import to my project. It resolves the type error.

jacobwgillespie commented:
Cool - yeah in your case, I think the act of importing them locally likely bumped the dependency versions in your lockfile. Currently the package specifies `*` as the version selector for peer dependencies which should allow for any version of those dependencies, but it's possible for package managers to end up having multiple versions of `kysely` when you upgrade the one your app is using, rather than upgrading both your app's version and this package's peer version.

If this happens again, you may want to try running `[npm|yarn|pnpm] dedupe`, to make the package manager unify your app's chosen `kysely` version with the peer dependency here.
